review previous portuguese docs (#17)
stack-spot/graphene-java-api-stack: tracing-plugin/docs/about.md

> ## General Vision

Reviewer: Sugestão 
Substituir general vision para **Overview**

> ### How the Tracing Plugin works

Reviewer: Se for uma pergunta, formular a frase utilizando o Does
How **does** the Tracing Plugin works

Author: Não é pergunta nesse caso

> ### How the Tracing Plugin works

Reviewer 2: ```suggestion
### How does the Tracing Plugin work?
```

> When applied, this plugin adds a class of configuration of _tracing_ making use of **`opentracing-spring-jaeger-web-starter`**.

Reviewer 2: ```suggestion
This plugin adds a _tracing_ configuration class using **`opentracing-spring-jaeger-web-starter`**.
```

> When start the application, automatically, it connects and sends _tracing_ to an local endpoint.

Reviewer 2: ```suggestion
When you start the application, it automatically connects and sends _tracing_ to a local endpoint.
```

> See more information about **jaeger** and how to deploy locally at [**jaegertracing**](https://www.jaegertracing.io/docs/1.37/deployment/)

Reviewer 2: ```suggestion
For more information about **Jaeger** and how to deploy locally, see the [**jaegertracing**](https://www.jaegertracing.io/docs/1.37/deployment/) page.
```
